packages/server/src/resources/user.rs

```diff
@@ -12,6 +12,7 @@ pub struct UserDetails {
 #[derive(Default, Clone, Debug, Deserialize, Serialize)]
```

Comment:
In addition, I tried to use the `dark_mode` locally, and there is an annoying state where after changing the mode, and refresh the page we get the pop up "סטטוס התואר שלך לא מעודכן...".